.snyk/dependency-sync.py

```diff
@@ -0,0 +1,15 @@
+from pathlib import Path
+
+import tomllib
+
+
+def sync():
+    pyproject = tomllib.loads(Path("pyproject.toml").read_text())
+    dependencies = pyproject["project"]["dependencies"]
```

Comment:
Are these all dependencies including dev deps or we have some development packages listed as dependencies?  
`coverage`

Reply:
Good point, I forgot about them